manager/state/raft/raft_test.go

```diff
@@ -975,7 +975,7 @@ func TestStreamRaftMessage(t *testing.T) {
 
 	raftMsg := &api.StreamRaftMessageRequest{Message: msg}
 	err = stream.Send(raftMsg)
-	assert.Error(t, err, "Received unexpected error EOF")
+	assert.NoError(t, err, "Received unexpected error EOF")
```

Comment:
Looks like the `"Received unexpected error EOF"` should be removed here?

Reply:
Ah sorry, I yes, thank you.

Reply:
Fixed